packages/video_player/video_player_web/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.0.6
+
+* Bump `video_player_platform_interface` version
```

Comment:
Why?

Reply:
wasn't sure about this change. I had problems with building example without this. But maybe that because there weren't versions constraints when I published pr.